src/Components/test/E2ETest/Tests/BindTest.cs

```diff
@@ -829,10 +829,10 @@ public void CanBindTextboxDateTimeOffsetWithFormat()
             Assert.Equal(expected, DateTimeOffset.Parse(boundValue.Text));
             Assert.Equal(expected, DateTimeOffset.Parse(mirrorValue.GetAttribute("value")));
 
-            // Clear textbox; value updates to emtpy because that's what we do for `default` when there's a format
+            // Clear textbox; value updates to the default
             target.Clear();
             expected = default;
-            Browser.Equal(string.Empty, () => target.GetAttribute("value"));
+            Browser.Equal("01-01", () => target.GetAttribute("value"));
```

Comment:
@SteveSandersonMS - let's discuss if you have concerns

This behaviour was inconsistent between the case where you have a format and where you don't. I made these behave the same, which broke the test. So for `default(DateTime)` we will tostring the value regardless of whether you specified a format. For `default(DateTime?)` we will return the empty string.

Reply:
I think that's perfect and matches exactly with the way we started treat numbers (i.e., clearing an `int`-bound input will revert to `0`).